README.adoc

```diff
@@ -422,6 +422,22 @@ run the following Maven command:
 ----
 ====
 
+If you use Gradle, you can append the following snippet to build.gradle
+====
+[source,text]
+----
+bootRun{
+	systemProperty 'server.port', '9000'
+}
+----
+====
+Then run the following Gradle command:
```

Comment:
Change to "Then you can run the following Gradle command:"

The idea is to add a sense of agency (the idea that people can do things).

Looks good otherwise.

Reply:
Thanks. Change implemented.